Source/RunActivity/Viewer3D/DDSLib.cs

```diff
@@ -516,15 +507,15 @@ private static int MipMapSizeInBytes(int map, int width, int height, bool isComp
         }
 
         //Get the byte data from a mip-map level.
-        private static void GetMipMaps(int offsetInStream, int map, bool hasMipMaps, int width, int height, bool isCompressed, FourCC compressionFormat, int rgbBitCount, bool partOfCubeMap, BinaryReader reader, LoadSurfaceFormat loadSurfaceFormat, ref byte[] data, out int numBytes)
+        private static void GetMipMaps(int offsetInStream, int map, bool hasMipMaps,int width, int height, bool isCompressed, FourCC compressionFormat, int rgbBitCount, bool partOfCubeMap, BinaryReader reader,LoadSurfaceFormat loadSurfaceFormat, ref byte[] data, out int numBytes)
```

Comment:
```suggestion
        private static void GetMipMaps(int offsetInStream, int map, bool hasMipMaps, int width, int height, bool isCompressed, FourCC compressionFormat, int rgbBitCount, bool partOfCubeMap, BinaryReader reader, LoadSurfaceFormat loadSurfaceFormat, ref byte[] data, out int numBytes)
```
There are some strange whitespace changes in this file--like this one, where some spaces between arguments were removed.

Reply:
These were from the original changes, so may have been automatically applied by Visual Studio when Peter originally did the migration.

I've applied [dotnet-format](https://github.com/dotnet/format) to the whole file (which we should adopt as our standard code style) to fix all these cases.